payment-handler/app-can-make-payment-false.js

```diff
@@ -0,0 +1,6 @@
+// Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang).
```

Comment:
I wonder if we can merge this file and the next one? Then we can send test conditions via the PMI: 

`https://w3c-test.org?canMakePayment=true`
`https://w3c-test.org?canMakePayment=false`
`https://w3c-test.org?canMakePayment=promise-false`
`https://w3c-test.org?canMakePayment=promise-true`

And so on...

Reply:
Merged the app files.